rust/processor/src/processors/econia_processor.rs

```diff
@@ -618,14 +651,76 @@ impl ProcessorTrait for EconiaTransactionProcessor {
                     )?);
                 }
             }
+            // Index transaction write set.
+            let info = &txn.info.as_ref().expect("No transaction info");
+            for change in &info.changes {
+                match change.change.as_ref().expect("No transaction changes") {
+                    Change::WriteResource(resource) => {
+                        if resource.r#type.as_ref().expect("No resource type")
+                            == &market_accounts_type
+                        {
+                            let data: serde_json::Value = serde_json::from_str(&resource.data)
+                                .expect("Failed to parse MarketAccounts");
+                            market_account_handles.push(MarketAccountHandle {
+                                user: resource.address.clone(),
+                                handle: data["map"]["handle"].as_str().unwrap().to_string(),
```

Comment:
See comment on https://github.com/econia-labs/econia/pull/511, I think we should include a time field here so that we can know what time an account was first made.

Reply:
Included as of 306a3cb